nipype/interfaces/io.py

```diff
@@ -322,6 +324,11 @@ def _list_outputs(self):
                 if isinstance(files[0], list):
                     files = [item for sublist in files for item in sublist]
 
+            config = NipypeConfig()
+            try_hard_link_datasink = config.get('execution',
```

Comment:
you can also use str2bool function from nipype utils:

`str2bool(config['execution']['try_hard_link_datasink'])`